### PR TITLE
Md to rst

### PR DESCRIPTION
**Pull request by greg0ire**
Using a single markdown file does not scale well. Also, it is less
suprising to do the same as in other Doctrine projects.

### CODE REVIEW

**Comment by greg0ire** [comment]
I suppose the next steps are to update https://github.com/doctrine/doctrine-website/blob/master/data/projects.yml, and then the README once we have a url.

**Review by derrabus — APPROVED** [review]

File: docs/en/how-to/fixture-ordering.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        public function getOrder(): int
+        {
+            return 10; // smaller means sooner
+        }
+    }
+
+.. note::
+    While extending ``AbstractFixture`` is not required, it is likely
+    you are going to need it since people usually need fixtures to be
+    loading in a specific order because of references form one fixture
```
Comment (stof) [inline docs/en/how-to/fixture-ordering.rst]: typo: `from`, not `form`

File: docs/en/how-to/fixture-ordering.rst [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    <?php
+
+    class MyFixture extends AbstractFixture implements DependentFixtureInterface
+    {
+        public function load(ObjectManager $manager): void
+        {
+        }
+
+        /**
+         * @return list<FixtureInterface>
```
Comment (stof) [inline docs/en/how-to/fixture-ordering.rst]: this is wrong. It is `list<class-string<FixtureInterface>>`

File: docs/en/how-to/loading-fixtures.rst [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+store you are using. For example, if you are using ORM, you should
+do the following:
+
+.. code-block:: php
+    <?php
+    use Doctrine\Common\DataFixtures\Executor\ORMExecutor;
+    use Doctrine\Common\DataFixtures\Purger\ORMPurger;
+
+    $purger = new ORMPurger();
+    $executor = new ORMExecutor($entityManager, new ORMPurger());
```
Comment (stof) [inline docs/en/how-to/loading-fixtures.rst:93]: shouldn't this use `$purger` ? If no, why defining it ?

Comment (greg0ire) [reply docs/en/how-to/loading-fixtures.rst:93]: Oh sorry, Noticed the one time variable, and forgot to delete line 93

**Pull request merged**